README.md

```diff
@@ -1 +1,67 @@
-# owasp-zap-ui-scan
\ No newline at end of file
+# OWASP ZAP UI Automated Scanning :zap:
+<p align="center"> <br> :exclamation: :exclamation:  <b> ONLY RUN THIS AGAINST APPLICATIONS YOU HAVE PERMISSION TO ATTACK </b> :exclamation: :exclamation: <br><br> </p>
+
+Provides the ability to execute a [Full Scan](https://github.com/zaproxy/zaproxy/wiki/ZAP-Full-Scan]) against a web application using the OWASP ZAP Docker image within a Azure DevOps pipeline. This generates the standard OWASP ZAP Html report and an NUnit test report to publish the results to the pipeline. 
+
+## Getting Started
+These instructions will enable you to get the Full Scan incorporated into an Azure DevOps pipeline. 
```

Comment:
Is Full Scan (with capitals) a thing? Lower case?

Reply:
It is... as per your comment below... will add the link here as well.